src/chi/components/marketing-icon/chi-marketing-icons.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
@mattnickles @jllr Do we need this file?
I believe it is based on `chi-icons.js` SVG injector that we supported up to `v1.4.2` https://assets.ctl.io/chi/1.4.2/components/icon/
and removed from the documentation starting from `v2.0.0`, though we still support `chi-icons.js` since it is in use in several projects.
If we don't document `chi-marketing-icons.js` I think we don't need it.